src/test/java/dev/openfeature/sdk/ValueTest.java

```diff
@@ -66,11 +66,11 @@ class Something {}
     }
 
     @Test public void numericArgShouldReturnDoubleOrInt() {
-        double innerDoubleValue = .75;
+        double innerDoubleValue = 1.75;
         Value doubleValue = new Value(innerDoubleValue);
         assertTrue(doubleValue.isNumber());
-        assertEquals(1, doubleValue.asInteger());     // should be rounded
-        assertEquals(.75, doubleValue.asDouble());
+        assertEquals(1, doubleValue.asInteger());     // the double value represented by this object converted to type int
```

Comment:
Thanks for linking the spec related to this rounding "toward zero". I was not aware of this nuance.